python/cudf/cudf/tests/test_s3.py

```diff
@@ -391,6 +391,49 @@ def test_read_parquet_filters(s3_base, s3so, pdf_ext, precache):
     assert_eq(pdf_ext.iloc[:0], got.reset_index(drop=True))
 
 
+def test_read_parquet_dask(s3_base, s3so, pdf):
+    dask_cudf = pytest.importorskip("dask_cudf")
+    dd = pytest.importorskip("dask.dataframe")
+
+    fname = "test_parquet_reader_dask.parquet"
+    bucket = "parquet"
+    buffer = BytesIO()
+    pdf.to_parquet(path=buffer)
+    buffer.seek(0)
+    with s3_context(s3_base=s3_base, bucket=bucket, files={fname: buffer}):
+        got = dask_cudf.read_parquet(
+            f"s3://{bucket}/{fname}",
+            storage_options=s3so,
+        )
+        dd.assert_eq(pdf, got)
+
+
+@pytest.mark.parametrize(
+    "options",
+    [
+        {"open_file_options": {"precache_options": {"method": "parquet"}}},
+        {"read": {"use_python_file_object": True}},
+    ],
+)
+def test_read_parquet_dask_warns(s3_base, s3so, pdf, options):
```

Comment:
Question:
Is there a reason we're putting the test in cudf as opposed to dask_cudf?
(I worry that these tests are at risk of getting skipped in CI)

There's a similar test to this one in python/dask_cudf/dask_cudf/io/tests/test_s3.py I think.
(but that one uses open_file_options - which is deprecated now - all the time, which I think is why we missed the noisy warnings)

Reply:
>There's a similar test to this one in python/dask_cudf/dask_cudf/io/tests/test_s3.py I think.

Ah - I put together this PR quickly and missed that we were already testing s3 somewhere in dask_cudf - I'll definitely move the tests.

Reply:
Okay - I moved these tests